collector/benchlib/src/benchmark.rs

```diff
@@ -172,34 +175,58 @@ mod tests {
 
     #[test]
     fn test_passes_filter_no_filter() {
-        assert!(passes_filter("foo", None, None));
+        assert!(passes_filter("foo", &[], &[]));
     }
 
     #[test]
     fn test_passes_filter_include() {
-        assert!(!passes_filter("foo", None, Some("bar")));
-        assert!(!passes_filter("foo", None, Some("foobar")));
-
-        assert!(passes_filter("foo", None, Some("f")));
-        assert!(passes_filter("foo", None, Some("foo")));
-        assert!(passes_filter("foo", None, Some("bar,baz,foo")));
+        assert!(!passes_filter("foo", &[], &["bar".to_string()]));
```

Comment:
Ugly.

Reply:
Arg is Vec<String>, so can't be converted to &[&str] inplace, requiring reallocating, except tests :-)

Reply:
Yeah, I know. I would still probably do it, since `&[&str]` is more natural to use. The reallocation isn't an important cost here. But let's keep it as it is for now.